src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -370,7 +384,9 @@ function MoneyRequestView({
                         }
                     />
                 )}
-                {canUseViolations && <ViolationMessages violations={getViolationsForField('receipt')} />}
+                {!(!hasReceipt && (canEditReceipt || isAdmin || isApprover)) && !(showMapAsImage || hasReceipt) && <View style={{marginVertical: 6}} />}
```

Comment:
lint issue

Reply:
This is impossible to read, please intermediate variables with self-explanatory names to simplify this